tests/greenheart/test_hydrogen/test_PEM_bop.py

```diff
@@ -0,0 +1,45 @@
+from pytest import approx, fixture
```

Comment:
I'd like to see one more test with the BOP in a system to make sure it is behaving as expected. Could be a quick small system.

Reply:
Okay I added a small integration test for the greenheart system. Let me know if that works